data/brands/shop/convenience.json

```diff
@@ -5798,6 +5798,18 @@
         "name:zh": "超市发",
         "shop": "convenience"
       }
+    },
+    {
+      "displayName": "十足",
+      "id": "shizu-zhejiang-cn",
```

Comment:
It is a pity that we currently do not have the geojson of China's provinces, so we cannot distinguish it. Also you don't need to manually add this id, we'll run build script everytime publish the package.

Reply:
Good to know. I wondered how to generate this id :)

Reply:
> Good to know. I wondered how to generate this id :)

Clone the whole repository and run `npm install` in project root dictionary, and run `npm run build` every time you've done some changes.

You can see maintainer chapter of project wiki: https://github.com/osmlab/name-suggestion-index/wiki/Info-for-Maintainers

Reply:
When I run `npm run build` many files changed. This pollutes my commit. I guess it's maintainers who should run it right?

Reply:
> When I run `npm run build` many files changed. This pollutes my commit. I guess it's maintainers who should run it right?

Yes, so I said you don't need to care about this 😃